mount/mountinfo_linux.go

```diff
@@ -45,10 +45,6 @@ func parseInfoFile(r io.Reader) ([]Info, error) {
 	out := []Info{}
 	var err error
 	for s.Scan() {
-		if err = s.Err(); err != nil {
-			return nil, err
-		}
-
```

Comment:
Same question as in the another your PR, how should help ignoring the most obvious and important err, scan error?

Reply:
according to `Scan` docs: https://golang.org/pkg/bufio/#Scanner.Scan

Scan advances the Scanner to the next token, which will then be available through the Bytes or Text method. `It returns false when the scan stops, either by reaching the end of the input or an error.` After Scan returns false, the Err method will return any error that occurred during scanning, except that if it was io.EOF, Err will return nil. Scan panics if the split function returns too many empty tokens without advancing the input. This is a common error mode for scanners.

seems missing check the `s.Err()` out of loop?